tests/kernel/kirk.pm

```diff
@@ -30,11 +36,31 @@ sub run
     my $suite = get_var('KIRK_SUITE', '');
 
     select_serial_terminal;
+    zypper_call("ref");
+
+    my $cmd = '';
+    if ($install_from =~ /git/i) {
+        if (is_transactional) {
+            trup_install("git");
+        } else {
+            zypper_call("in -y git");
+        }
```

Comment:
I would do one liner `is_transactional ? trup_install("git") : zypper_call("in -y git");`, but is just different style.